### PR TITLE
This weekend's Efforts

### PR DESCRIPTION
**Pull request by sumps**
Fixed some link issues, added a couple of new Gallery images and tested and then published a new blog post, which I had not done before.

### CODE REVIEW

File: _posts/2020-04-26-recent-articles.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+status: published
+---
+
+The message is gradually getting out there about the benefits of Signal K and there have been some really interesting articles published recently on other blogs, and we thought we would give you links to them here.
+
+### Victron Blog - Sailing with Signal K
+
+Victron the innovative and highly respected Dutch manufacturer of energy products has embraced Signal K and this post tells the story of sailing boat Taku Moe and how the owners Stefanie and Heike have installed and used Signal K with their Victron system.
+
+[https://www.victronenergy.com/blog/2019/06/13/sailing-with-signal-k/](https://www.victronenergy.com/blog/2019/06/13/sailing-with-signal-k/)
```
Comment (tkurki) [inline _posts/2020-04-26-recent-articles.md:16]: While a cool entry this is now close to a year old and to me hardly counts as _recent_. Something else instead?

File: _posts/2020-04-26-recent-articles.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+### Sailbits.com - Real-Time Signal K data  
+
+Steve Mitchell, a long time Champion of Signal K has published a great article on how he remotely accessed real-time weather data from his boat using Signal K.
+
+[https://seabits.com/real-time-weather-from-the-boat/](https://seabits.com/real-time-weather-from-the-boat/)
+
+### Henri Bergius - Signal K refit  
+
+Acive sailor, adventurer and software engineer, Henri Bergius has written an excellent article on his Signal K installation.
```
Comment (tkurki) [inline _posts/2020-04-26-recent-articles.md:26]: Typo in active

File: _posts/2020-04-26-recent-articles.md (hunk 1 above)
Comment (tkurki) [reply _posts/2020-04-26-recent-articles.md:16]: Panbo article?
Or 
https://bareboat-necessities.github.io/my-bareboat/

Comment (sumps) [reply _posts/2020-04-26-recent-articles.md:16]: In an ideal world the Victron article would have been posted in our blog at the same time Victron published it but it wasn’t. I felt it was important enough to still have it in our blog even if it is not bang up to date.

I could swap it for the bare necessities post in this “recent” blog post and create a back dated post on the Victron article, I think it justifies this and should be recorded on our site.

Comment (tkurki) [reply _posts/2020-04-26-recent-articles.md:16]: i don’t think our blog is such a reference that backdating is needed, swap it for barenecessities and let’s move forward?

**Comment by sumps** [comment]
Have made the changes you requested and added two Yacht Devices' gateways that are now CANBoatJS compatible as requested by Scott. Please review and merge.

**Pull request merged**